tests/local-testnet/deploy.sh

```diff
@@ -17,6 +17,12 @@ if [ -z $EXECUTION ]; then
   EXECUTION=wasm
 fi
 
+echo "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
+echo "!!        Don't runtime thie script multiple times        !!"
```

Comment:
`Don't run thie script multiple times` ?

Reply:
```
time(s) |node processes
1        4
2        8
3        12
...
```

You should kill all the previous processes before next running.

Reply:
Hmm, I mean there's something wrong with that sentence, it should be `run` instead of `runtime`.

Reply:
Damn. cc @GitHubCopilot